### ISSUE TITLE
Spaces after commands in "Custom key mappings"

### ISSUE THREAD

**Issue by kapsh** [philc/vimium#1443]
Hello! 
If you accidentally put space after command in textarea "Custom key mappings" in options, that command won't be parsed (e.g. "map л scrollUp " instead of "map л scrollUp"). I think that it is the defect which can be fixed trivially.

**Comment by smblott-github** [comment]
Hi @kapsh.  Thanks for the report.  I can reproduce this in 1.42 (which is quite old), but not in 1.49 (the most recent release) or `master`.

Could you double check, please?

**Comment by kapsh** [comment]
Hello, 
I checked this one more time. Let me show you a couple of screenshots. 
1. https://i.imgur.com/Ov0zzOL.png 
At first you can see excess space after scrollDown - so this mapping isn't displayed in Help window (I pressed Save changes for sure).
2. https://i.imgur.com/OiYp2d3.png
At second there are no excess space AND new mapping is displayed in Help (and it works of course).

I am using version 1.49 from Chrome Web Store: https://i.imgur.com/beCD3YA.png

**Comment by z0rch** [comment]
I can actually confirm this on the latest `master`, with a single addition: a trailing space breaks the mapping **only if it isn't the latest mapping**.

Examples (`_` would represent a space):

If this is the entire configuration: 
```
map Q scrollLeft_
``` 

`Q` is mapped correctly.

However, with the following configuration:
```
map Q scrollLeft_
map W scrollRight_
```

`W` is mapped, but `Q` is not. 

---------

I was testing on Windows, if needed, I can verify the behavior on Linux in a few hours.

**Comment by kapsh** [comment]
Only, are you kidding? Most of mappings aren't latest mappings! :)Unfortunatelly, I cannot help you with the code. I am even can't understand why [https://github.com/philc/vimium/blob/master/pages/options.coffee#L63](this trim()) don't work. -- Александр Капшуна+7-968-65-46-276kapsh@kap.sh   28.01.2015, 16:31, "Maxim Baz" <notifications@github.com>:I can actually confirm this on the latest master, with a single addition: a trailing space breaks the mapping only if it isn't the latest mapping.Examples (_ would represent a space):If this is the entire configuration:map Q scrollLeft_
Q is mapped correctly.However, with the following configuration:map Q scrollLeft_
map W scrollRight_
W is mapped, but Q is not.I was testing on Windows, if needed, I can verify the behavior on Linux in a few hours.—Reply to this email directly or view it on GitHub.

**Comment by z0rch** [comment]
> Most of mappings aren't latest mappings! :)

Precisely.

> ... trailing space breaks the mapping only if it **isn't** the last mapping ...

In other words, "trailing space does not break the last mapping, but breaks everything else". That can also be seen from the examples I provided.

This was probably the reason @smblott-github could not reproduce the bug. If he had only 1 line in his configuration, trailing space does not break it. So you need to set at least 2 mappings to repro this issue.

**Comment by smblott-github** [comment]
> This was probably the reason @smblott-github could not reproduce the bug.

Yup!

Thanks for this @kapsh.  It's fixed in #1448.  It turns out...

```
_map a scrollDown
```

doesn't work either.  I've left that as is, for now.

**Comment by kapsh** [comment]
Cool, thank you! 

_map a command isn't necessary to fix I think: this mistake is much more obvious and can be seen easier.
